string jpos
string jvel
string jtor

string cpos
string[3] crot

string ctrl
string space
string pos

int32 gcomp
int32 holding
int32 teaching
string loadedRefgen
string activeRefgen
int32 moveIsDone
